src/WinRT.Runtime/Projections/INotifyPropertyChanged.net5.cs

```diff
@@ -2,31 +2,36 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Reflection;
 using System.Runtime.InteropServices;
 using WinRT;
+using WinRT.Interop;
 
 namespace ABI.System.ComponentModel
-{
+{
 #if EMBED
     internal
 #else
     public
 #endif
     static class INotifyPropertyChangedMethods
-    {
-        public static global::System.Guid IID { get; } = new Guid(new global::System.ReadOnlySpan<byte>(new byte[] { 0x01, 0x76, 0xB1, 0x90, 0x65, 0xB0, 0x6E, 0x58, 0x83, 0xD9, 0x9A, 0xDC, 0x3A, 0x69, 0x52, 0x84 }));
+    {
+        public static global::System.Guid IID { get; } = GuidGenerator.GetWuxMuxIID(typeof(INotifyPropertyChanged).GetCustomAttribute<WuxMuxProjectedTypeAttribute>());
```

Comment:
I don't really love this bit of performance hit on NativeAOT (I don't think it'll be able to correctly preinitialize everything). Can we replace all of these `IID` initialization with a helper method that simply returns the right value based on the feature switch? Then we can still keep both in RVA fields, and the linker will just drop the one that's unused.

I mean something like:
```csharp
public static global::System.Guid IID { get; } = new(GetIID());

private static ReadOnlySpan<byte> GetIID()
{
    return FeatureSwitches.IsMicrosoftUIXaml
        ? new global::System.ReadOnlySpan<byte>(new byte[] { ... })
        : new global::System.ReadOnlySpan<byte>(new byte[] { ... });
}
```

Reply:
I was just trying to avoid typing out the same guids a billion times, but I can manually copy them I guess.

Reply:
Where else do we have to copy them, and why can we not just reference this `IID` field from there? Do you just mean the copies as in, the string versions that are literal expressions for the attribute arguments?

Reply:
Yeah mainly that. There's also some duplication on the delegate types that's harder to get rid of, but that's okay. I'll clean it up.